src/routes/comments/index.ts

```diff
@@ -3,20 +3,40 @@
 import * as awsx from '@pulumi/awsx';
 import { v4 as uuidv4 } from 'uuid';
 import { DynamoDB } from 'aws-sdk';
-import axios from 'axios';
-import { marshall, unmarshalls, parseBody, buildApiResponse, add500Handler } from '$src/apiGatewayUtilities';
+import { marshall, parseBody, buildApiResponse, add500Handler } from '$src/apiGatewayUtilities';
 import { createDynamo } from '$src/initAWS';
+import AnalysisService from '$src/services/AnalysisService';
 
 import { getMoodType } from './moodTypeConversion';
 
+function toResponse(entry: DynamoDB.AttributeMap) {
+  return {
+    id: entry.id.S,
+    text: entry.text.S,
+    propertyId: entry.propertyId.S,
+    author: {
+      id: entry.author.M!.id.S,
+      firstName: entry.author.M!.firstName.S,
+      lastName: entry.author.M!.lastName.S,
+      avatarUrl: entry.author.M!.avatarUrl.S,
+    },
+    moodType: entry.mood.M!.type.S,
+    createdDate: entry.createdDate.S,
+  };
+}
+
+function unmarshalls(items: DynamoDB.Types.ItemList) {
```

Comment:
lets rename this to somethign like toArrayResponse or something else more specific, the unmarshall name can be mistakenly confused with unmarshall from AWS.DynamoDB library